course_discovery/settings/base.py

```diff
@@ -71,9 +71,9 @@
     'course_discovery.apps.publisher',
     'course_discovery.apps.publisher_comments',
     'course_discovery.apps.journal',
+    'course_discovery.apps.edly',
 ]
 
-
```

Comment:
this diff can be removed

Reply:
This diff was created due to adding new line `course_discovery.apps.edly` in `PROJECT_APPS`